.circleci/config.yml

```diff
@@ -65,6 +74,9 @@ jobs:
       - setup_build_environment
       - setup_remote_docker
       - populate_and_save_cache
+      - run:
```

Comment:
Not necessary, but it'd be nice to move this into its own command rather than duplicating in each job (see: https://github.com/hypertrace/attribute-service/pull/28/files )

Reply:
done.